tasks/InsightAppSec/helpers/insightAppSecApi.ts

```diff
@@ -316,7 +316,9 @@ export default class InsightAppSecApi
                     if (xhr.status < 200 || xhr.status > 299) {
                         console.error("Failed to return valid response from InsightAppSec API; Status Code: " + xhr.status +
                             ". Please Contact Rapid7 Support if this continues to occur.");
+                        console.error(xhr.responseText);
```

Comment:
Could we add context around the "response text" for what it is simply for easy understand for someone in support, etc? Should we add it to line above?

Reply:
Yep, will do